packages/augur-ui/src/modules/reporting/common.tsx

```diff
@@ -50,14 +60,28 @@ export const DismissableNotice = (props: DismissableNoticeProps) => {
           <span>
             {ExclamationCircle}
           </span>
-          {props.content}
-          <button
-            type='button'
-            className={Styles.close}
-            onClick={() => setShow(() => false)}
-          >
-            {XIcon}
-          </button>
+          <div>
+            <div>{props.title}</div>
+            <div>{props.description}</div>
+          </div>
+          {props.buttonType === DISMISSABLE_NOTICE_BUTTON_TYPES.BUTTON &&
+            <button
+              type='button'
+              className={ButtonStyles[props.buttonType]}
+              onClick={props.buttonAction}
+            >
+              {props.buttonText}
+            </button>
+          }
+          {props.buttonType === DISMISSABLE_NOTICE_BUTTON_TYPES.CLOSE &&
+            <button
+              type='button'
+              className={Styles[props.buttonType]}
```

Comment:
i think you can just use buttons here the normal way...?

Reply:
Yeah, I'll just use Styles.PrimaryButton and Styles.close